src/web/src/theme/index.ts

```diff
@@ -21,15 +21,15 @@ export const lightTheme: Theme = createMuiTheme({
       main: red.A400,
     },
     background: {
-      default: '#E5E5E5',
+      default: '#FFFFFF',
       paper: '#d1d1d1',
     },
     text: {
       primary: '#000000',
       secondary: '#121D59',
     },
     action: {
-      selected: '#ffffff',
+      selected: '#FFFFFF',
```

Comment:
What does `action.selected` mean?  When is it used, and should it be different from the regular background or the same?

Reply:
this is due to @izhuravlev commit earlier for the search page. so when you click on a filter, it made the selected filter white box. I was surprised to but did not want to remove it for now.

Reply:
Action: selected should be different from the background. Search bar changes it's color on hover to theme.palette.action.selected. 
I added this so we can set up 2 different colours for both modes.

Reply:
To make sure everything looks good, click or hover on the search bar in the light mode to see how it looks.